games-util/gamemode/gamemode-1.5.1.ebuild

```diff
@@ -0,0 +1,148 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit meson multilib-minimal ninja-utils systemd
+
+DESCRIPTION="Optimise Linux system performance on demand"
+HOMEPAGE="https://github.com/FeralInteractive/gamemode"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/FeralInteractive/gamemode.git"
+	GAMEMODE_GIT_PTR="master"
+	inherit git-r3
+else
+	GAMEMODE_GIT_PTR="${PV}"
+	SRC_URI="https://github.com/FeralInteractive/gamemode/releases/download/${GAMEMODE_GIT_PTR}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="systemd elogind"
+
+REQUIRED_USE="^^ ( systemd elogind )"
+
+RDEPEND="
+	elogind? ( sys-auth/elogind )
```

Comment:
Hmm, I think we can drop this? The usedep on dbus should pull it in, shouldn't it?

Reply:
I just dropped it.